### PR TITLE
Add missing function documentation, clean up comments

### PR DESCRIPTION
**Pull request by macedo22**
## Checklist

- [N/A] Did you write tests for your code?
- [ ] Did all the tests pass locally?

## Trello Ticket

https://trello.com/c/I0ePnKoV/96-add-header-to-all-functions-no-ctor-and-render

## Short Description

What does this PR contain?

This PR fills in missing function documentation and cleans up comments.

## Reviewer Notes

Anything special you want the reviewer to know?

Please see my comments that I made in the changes below.

### CODE REVIEW

File: src/modules/Login.js [hunk 1]
```diff
@@ -22,28 +22,31 @@ class Login extends React.Component {
...
-            displayName: user.displayName,
-          };
-          // User is signed in
-          this.loginWithGoogleUserAndRedirect(googleUser);
-        } else {
-          // Not signed in
-          this.setState({ loading: false });
-        }
-      });
+    this.authListener = firebase.auth().onAuthStateChanged((user) => {
```
Comment (macedo22) [inline src/modules/Login.js:30]: Note that the motivation for renaming this was that when I was reading this function to try to determine a header, it didn't seem like it was something for unsubscribing a listener. But I didn't write this, so let me know if I should switch it back.

File: src/modules/Login.js [hunk 2]
```diff
@@ -22,28 +22,31 @@ class Login extends React.Component {
...
+        this.setState({ loading: false });
+      }
+    });
 
+    // for cycling through potato jokes on login page
     this.timer = setInterval(this.incrementImgState, 6000);
-  }
+  } /* componentDidMount */
 
   componentWillUnmount() {
```
Comment (macedo22) [inline src/modules/Login.js:48]: I think this is the only remaining function that needs to be documented, but I didn't know what the description would be. If anyone has a suggestion, let me know and I'll add it

**Review by macedo22 — COMMENTED** [review]
I started updating our missing documentation, but when I got to `Login.js`, I ended up not feeling very confident about understanding a couple things, so please take a close look at the changes and documentation I made there, and let me know how I should document `componentWillUnmount` for it

File: src/modules/Login.js (hunk 2 above)
Comment (Joyfal) [reply src/modules/Login.js:50]: "Unsubscribes Firebase Auth listener, clears landing page picture shuffling timer"

**Review by Joyfal — APPROVED** [review]
LGTM, I gave you a description of what the componentwillunmount function do.

**Review by Joyfal — APPROVED** [review]
LGTM!

**Pull request merged**